packages/expo-constants/src/Constants.types.ts

```diff
@@ -293,6 +293,7 @@ export interface NativeConstants {
    */
   systemFonts: string[];
   systemVersion?: number;
+  supportedExpoSdks?: string[];
```

Comment:
let's mark this as private and be sure to leave it out of documentation

Reply:
So maybe adding this was a mistake? 😅

But I can see this array in managed workflows and in Snack, so for now I'm going to hide it and remove the changelog entry for this addition.